src/BenchmarkDotNet/Templates/WasmCsProj.txt

```diff
@@ -0,0 +1,43 @@
+﻿<Project Sdk="$SDKNAME$" DefaultTargets="BuildApp">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <OutputPath>bin</OutputPath>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+    <TargetFramework>$TFM$</TargetFramework>
+    <TargetArchitecture>wasm</TargetArchitecture>
+    <TargetOS>Browser</TargetOS>
+    <RuntimePackDir>$RUNTIMEPACKDIR$</RuntimePackDir>
+    <AppDir>$(MSBuildThisFileDirectory)\bin\net5.0\browser-wasm\publish</AppDir>
```

Comment:
if `$TFM$` is `net5.0`, we should use it everywhere (to allow for using `net6.0` in the future)

```suggestion
    <AppDir>$(MSBuildThisFileDirectory)\bin\$TFM$\browser-wasm\publish</AppDir>
```

Reply:
@naricc it doesn't look like Adam's comment was resolved here.